.github/workflows/jira-lint.yaml

```
@@ -13,4 +19,4 @@ jobs:
           jira-base-url: https://tyktech.atlassian.net/
           skip-branches: '^(release-[0-9.-]+(lts)?|master|main)$'
           validate_issue_status: true
-          allowed_issue_statuses: "In Dev, In Code Review, Ready For Testing, In Test, In Progress, In Review"
\ No newline at end of file
+          allowed_issue_statuses: "In Dev,In Code Review,Ready For Testing,In Test,In Progress,In Review"
```

```suggestion
          allowed_issue_statuses: "In Dev,In Code Review,Ready For Testing,In Test,In Progress,In Review"
          
```

add trailing newline, removing github warning

---

merged